torchci/lib/bot/cliParser.ts

```diff
@@ -0,0 +1,83 @@
+import minimist from "minimist";
+
+const cliOptions: { [key: string]: any } = {
+    revert: {
```

Comment:
now that we're here, do we want to make the user provide more than a message?

e.g., do we want to enforce both a reason and a link?